### PR TITLE
Adding Customer.Create 

### PR DESCRIPTION
**Pull request by davidtonghelix**

### CODE REVIEW

File: oauth.go [hunk 1]
```diff
@@ -1,14 +1,14 @@
 package goshopify
```
Comment (gir) [inline oauth.go]: I would revert this file.

**Review by gir — APPROVED** [review]

File: oauth.go (hunk 1 above)
Comment (davidtonghelix) [reply oauth.go]: Somehow I wasn't able to. I did `git checkout origin/master oauth.go` and confirmed it got reset. When I did git push again this file shows up.

Comment (davidtonghelix) [reply oauth.go]: Had to close my IDE to prevent auto-formatting even on external file change

Comment (gir) [reply oauth.go]: It actually might have been a git hook from devtools.

**Pull request merged**